spring-boot-project/spring-boot-docs/src/docs/asciidoc/features/sql.adoc

```diff
@@ -322,6 +322,21 @@ You can customize the console's path by using the configprop:spring.h2.console.p
 
 
 
+[[features.sql.h2-web-console.spring-security]]
+==== Configuring Spring Security for H2 Console
```

Comment:
I am slightly worried that we had something in the reference guide that essentially gives access to everyone to a resource, especially with the current naming.